shared/src/main/scala/profig/ConfigPath.scala

```diff
@@ -133,10 +132,6 @@ class ConfigPath(val path: List[String]) {
         case Right(value) => value
       }
       combine(json, defaults)
-    } else {                                // Properties?
```

Comment:
You should update the ScalaDoc for this method as well if we can't support Properties in it.